src/dotnet/CommandLine/CommandLineApplication.cs

```diff
@@ -64,7 +64,21 @@ public CommandLineApplication Command(string name, bool throwOnUnexpectedArg = t
         public CommandLineApplication Command(string name, Action<CommandLineApplication> configuration,
             bool throwOnUnexpectedArg = true)
         {
-            var command = new CommandLineApplication(throwOnUnexpectedArg) { Name = name, Parent = this };
+            var command = new CommandLineApplication(throwOnUnexpectedArg) { Name = name };
+            return Command(command, configuration, throwOnUnexpectedArg);
+        }
+
+        public CommandLineApplication Command(CommandLineApplication command, bool throwOnUnexpectedArg = true)
+        {
+            return Command(command, _ => { }, throwOnUnexpectedArg);
+        }
+
+        public CommandLineApplication Command(
+            CommandLineApplication command,
+            Action<CommandLineApplication> configuration,
+            bool throwOnUnexpectedArg = true)
```

Comment:
It's good policy to have null argument checks on these so that if/when a null is passed the exception is thrown at the point where the coding error was made rather than on some later invocation.